src/main/java/de/tum/in/www1/artemis/web/rest/ModelingExerciseResource.java

```diff
@@ -211,10 +210,6 @@ public ModelingExerciseResource(ModelingExerciseRepository modelingExerciseRepos
     @PreAuthorize("hasAnyRole('ADMIN')")
     public ResponseEntity<Void> getCompassStatisticForExercise(@PathVariable Long exerciseId) {
         ModelingExercise modelingExercise = modelingExerciseService.findOne(exerciseId);
-        if (!authCheckService.isAdmin()) {
```

Comment:
I think this method is not necessary since all users who do not have the user role `ADMIN` cannot access this endpoint (`@PreAuthorize("hasAnyRole('ADMIN')")`)